### ISSUE TITLE
[CRASH] WingetUI 3 is blocked by Crowdstrike

### ISSUE THREAD

**Issue by bjrnb** [marticliment/WingetUI#1951]
### Please confirm these before moving forward

- [X] I have tried deleting a folder named `.wingetui` placed on the user directory
- [X] I have tried reinstalling WingetUI
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).

### Describe your crash

Hi,

I am using WingetUI on my Business-PC, which is protected by SIEM Solution "Crowdstrike". 

After the Update to 3.0, WingetUI will be closed due to "Reflective Code Loading"

![crowdstrike-blocking-wingetui](https://github.com/marticliment/WingetUI/assets/109213956/130c1af6-008c-4d8b-b46f-6e6562da0256)

This is even triggering an alert to the infrastructure team as part of a high alert case.

I am not able to provide more information from the crowdstrike backend due to a holiday of a colleague. Will provide more information if this colleague is available again.

So I reinstalled WingetUI 3, nothing changed.

Now I am running on 2.X again.

**Comment by marticliment** [comment]
Duplicate of #1927
